src/components/application_manager/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -199,6 +217,71 @@ TEST_F(OnSystemRequestNotificationTest, Run_RequestNotAllowed_NoNotification) {
   command->Run();
 }
 
+TEST_F(
+    OnSystemRequestNotificationTest,
+    Run_RequestTypeAllowedAndRequestSubTypeDisallowed_MessageNotSentToMobile) {
+  MessageSharedPtr msg = CreateMessage();
+  (*msg)[strings::params][strings::connection_key] = kConnectionKey;
+  const auto request_type = mobile_apis::RequestType::HTTP;
+  (*msg)[strings::msg_params][strings::request_type] = request_type;
+  const std::string request_subtype = "fakeSubType";
+  (*msg)[am::strings::msg_params][am::strings::request_subtype] =
+      request_subtype;
+
+  PreConditions();
+
+  EXPECT_CALL(mock_policy_handler_,
+              IsRequestTypeAllowed(kPolicyAppId, request_type))
+      .WillOnce(Return(true));
+  EXPECT_CALL(mock_policy_handler_,
+              IsRequestSubTypeAllowed(kPolicyAppId, request_subtype))
+      .WillOnce(Return(false));
+
+  EXPECT_CALL(app_mngr_, SendMessageToMobile(_, _)).Times(0);
+
+  auto command = CreateCommand<OnSystemRequestNotification>(msg);
+
+  ASSERT_TRUE(command->Init());
+  command->Run();
+}
+
+TEST_F(OnSystemRequestNotificationTest,
+       Run_RequestTypeAllowedAndRequestSubTypeAllowed_SendMessageToMobile) {
+  MessageSharedPtr msg = CreateMessage();
+  (*msg)[strings::params][strings::connection_key] = kConnectionKey;
+  const auto request_type = mobile_apis::RequestType::OEM_SPECIFIC;
+  (*msg)[strings::msg_params][strings::request_type] = request_type;
+  const std::string request_subtype = "fakeSubType";
```

Comment:
@BSolonenko Your new test second time use "const std::string request_subtype = "fakeSubType";",
It would be nice declare this variable in global namespace near "kPolicyAppId", wouldn't?